.pre-commit-config.yaml

```diff
@@ -141,7 +141,7 @@ metastore_browser/templates/.*\\.html$|.*\\.jinja2"
     hooks:
       - id: check-hooks-apply
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.5.0
+    rev: v3.1.0
```

Comment:
> pre-commit/pre-commit-hooks now requires python3.6.1+

That is the only noticeable difference for us.

Ref: https://github.com/pre-commit/pre-commit-hooks/releases